core-bundle/src/EventListener/DataContainer/PageUrlListener.php

```diff
@@ -287,6 +290,29 @@ private function aliasExists(string $currentAlias, int $currentId, PageModel $cu
         return false;
     }
 
+    private function addInputToPage(PageModel $pageModel): void
+    {
+        $input = $this->framework->getAdapter(Input::class);
+
+        if (null !== ($type = $input->post('type'))) {
+            $pageModel->type = $type;
+        }
+
+        if ('root' === $pageModel->type) {
+            if (null !== ($dns = $input->post('dns'))) {
+                $pageModel->domain = $dns;
+            }
+
+            if (null !== ($urlPrefix = $input->post('urlPrefix'))) {
+                $pageModel->urlPrefix = $urlPrefix;
+            }
+
+            if (null !== ($urlSuffix = $input->post('urlSuffix'))) {
+                $pageModel->urlSuffix = $urlSuffix;
+            }
+        }
```

Comment:
Should we add `$pageModel->preventSaving()` here just to be safe?

Reply:
no, this would create another copy of the page even though they already are through `loadDetails`

Reply:
I don't think so. Its a private method, we're the only one calling it and just need to make sure. But I actually fixed/improved that in b70f907495de097e6b124bb294752e01deaa1bcb

Reply:
`preventSaving` is public, but with `findWithDetails` `preventSaving` is already called now, yes 👍